wgsl/index.bs

```diff
@@ -11310,19 +11310,19 @@ Must only be invoked in [=uniform control flow=].
 ```rust
 textureSample(t: texture_1d<f32>, s: sampler, coords: f32) -> vec4<f32>
 textureSample(t: texture_2d<f32>, s: sampler, coords: vec2<f32>) -> vec4<f32>
-textureSample(t: texture_2d<f32>, s: sampler, coords: vec2<f32>, offset: vec2<i32>) -> vec4<f32>
-textureSample(t: texture_2d_array<f32>, s: sampler, coords: vec2<f32>, array_index: i32) -> vec4<f32>
-textureSample(t: texture_2d_array<f32>, s: sampler, coords: vec2<f32>, array_index: i32, offset: vec2<i32>) -> vec4<f32>
+textureSample(t: texture_2d<f32>, s: sampler, coords: vec2<f32>, offset: vec2<u32>) -> vec4<f32>
```

Comment:
I'm confused. Offsets can be negative.

Reply:
Good catch, got a little overzealous with search and replace.